roscomm/stringlist[] tables
roscomm/intlist[] current_selection
int32[] selection_mode
int32   last_selected_item